delivery/common/jwt/lib/jwt.d.ts

```diff
@@ -0,0 +1,117 @@
+/**
+ * JWTJson class to hold JWT header and payload in JSON format
+ */
+declare type JWTJson = {
+    /**
+     * JWT header in JSON format. see {@link JWTHeader} for more details
+     */
+    header: JWTHeader;
+    /**
+     * JWT payload in JSON format. see {@link JWTPayload} for more details
+     */
+    payload: JWTPayload;
+};
+declare type JWTHeader = {
+    alg: string;
+    typ: string;
+};
+declare type JWTPayload = {
```

Comment:
Style: It's up to you, but including the first sentence or two from the RFC for each field might be nice for usability. See https://www.rfc-editor.org/rfc/rfc7519.html#section-4.1